src/main/java/io/confluent/connect/jdbc/util/JdbcUtils.java

```diff
@@ -243,7 +243,7 @@ public static Timestamp getCurrentTimeOnDB(
     String dbProduct = conn.getMetaData().getDatabaseProductName();
     if ("Oracle".equals(dbProduct)) {
       query = "select CURRENT_TIMESTAMP from dual";
-    } else if ("Apache Derby".equals(dbProduct) || "DB2 UDB for AS/400".equals(dbProduct)) {
+    } else if ("Apache Derby".equals(dbProduct) || dbProduct.startsWith("DB2")) {
```

Comment:
Should we be super cautious and add: 
`(dbProduct != null && dbProduct.startsWith("DB2"))` instead?

A matter of style more than anything else, rather than a possibility of an implementation of `getDatabaseProductName` ever returning `null`